### PR TITLE
Developer guide for edit, budget, delete

### PR DESCRIPTION
**Pull request by rharwo**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -120,10 +128,20 @@ To add new entries, user will have to input the related commands.
...
-#### Deleting of entry, editing of entry
-The commands are similar where there would be a parsing of command to each of its individual classes.
-A similar validation process takes place and actions would be made on IncomeList or SpendingList accordingly
-(deleting entry from list for delete and editing of entry from list for edit)
+#### Editing entries
+The EditCommand validates and parses the given input to determine if it is editing a spending or an income. It then
+extracts the entry from either the respective list(SpendingList or IncomeList). Finally, it uses the parsed input to
+determine which attribute to edit and sets this attribute of the extracted entry to the new value.
+
+![editCommandSequence.png](./Diagrams/Commands/editCommandSequence.png)
```
Comment (TPH777) [inline docs/DeveloperGuide.md:136]: The alt frame should have dotted line in between

**Review by TPH777 — COMMENTED** [review]
Overall lgtm! Just a minor mistake

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (rharwo) [reply docs/DeveloperGuide.md:136]: Fixed!

**Comment by TPH777** [comment]
lgtm!

**Pull request merged**